assets/fetch-normalized-vms.sql.j2

```diff
@@ -0,0 +1,44 @@
+#standardsql
+WITH
+
+###
+### Apply basic transformations to the Norway VMS positions.
+###
+raw_vms_positions_normalized AS (
+  SELECT
+    message_id as msgid,
+    `world-fishing-827.udfs_v20210701.normalize_shipname`(vessel_name) as shipname,
+    timestamp_utc as timestamp,
+    lat,
+    lon,
+    speed,
+    course,
+    concat(`world-fishing-827.udfs_v20210701.normalize_shipname`(vessel_name), '-', callsign) as ssvid,
+    IF(callsign != "", callsign, NULL) callsign,
+    CASE STARTS_WITH(UPPER(vessel_type),'FISKEFART')
+      WHEN TRUE THEN 'Fiskefartøy'
+      WHEN FALSE THEN vessel_type
+    END as shiptype,
+    CONCAT("norway_vms_", CASE STARTS_WITH(UPPER(vessel_type),'FISKEFART')
+      WHEN TRUE THEN 'fiskefartøy'
+      WHEN FALSE THEN LOWER(vessel_type)
+    END) as source,
```

Comment:
Why are you contacting the vessel_type here? Other countries we only have VMS_Country. Chile is the only different one, because we receive 4 different feeds of information.

Reply:
+1, this should be `VMS_Norway`. Remember @smpiano that source is a field we use to mark distinct data sources. Norway has only one data source so far. It has multiple vessel types in that source, but it's still only one source.

Reply:
I assumed wrong using Costa Rica as template.
Costa Rica and Chile, both has dependency of the fleet.
Panama is `panama_vms`.
Brazil is `ONYXSAT_BRAZIL_VMS`.
Peru are `peru.historic` and `peru.trasat_api`.
Ecuador is `ecuador_vms`.
Mexico is `mexico`.
@enriquetuya @andres-arana Do you still agree that should it be `VMS_Norway` or should be `norway_vms`?

Reply:
ugh... what a mix.. Let's use `norway_vms`, 3 of them has `vms` as postfix and none as a prefix.

Reply:
`norway_vms` is fine. Eventually, it would be nice to have a standard convention, something like `[COUNTRY]_vms_[SUBSOURCE]`. So `chile_vms_aquaculture`, `brazil_vms_onyxsat` or `peru_vms_historic`. Countries with no subsources would be `[COUNTRY]_vms`, like `ecuador_vms`.